rc-events-service/src/main/scala/com/epam/random_coffee/events/api/request/CreateEventRequest.scala

```diff
@@ -0,0 +1,3 @@
+package com.epam.random_coffee.events.api.request
+
+case class CreateEventRequest(eventName: String)
```

Comment:
[OPT]
Mb if we use that field in `CreateEventRequest`, it's obviously, that it's event name
```suggestion
case class CreateEventRequest(name: String)
```

Reply:
changed